doc/classes/StringName.xml

```diff
@@ -954,18 +954,24 @@
 			<return type="String" />
 			<param index="0" name="right" type="Variant" />
 			<description>
+				Formats the [StringName], replacing the placeholders with one or more parameters.
+				To pass multiple parameters, [param right] needs to be an [Array].
+				In C#, there is no direct equivalent to this operator. Use the [method format] method, instead.
+				For more information, see the [url=$DOCS_URL/tutorials/scripting/gdscript/gdscript_format_string.html]GDScript format strings[/url] tutorial.
```

Comment:
Do we really want to have 4 paragraphs here?

Reply:
Not going to merge the "In C#..." and "For more information..." together. It would read as "to read more about C#'s `format()` method, check the tutorial", which is not fully true,